merge-proto.sh

```diff
@@ -0,0 +1,107 @@
+#! /bin/bash
+
+GREEN='\e[0;32m'
+YELLOW='\e[1;33m'
+RESET='\e[0m'
+RED='\e[1;31m'
+BLUE='\e[0;34m'
+
+: ${TMP_DIR:="./tmp"}
+: ${DST_FILE:="./generated/fig/tendermint/codec/v1/codec.proto"}
```

Comment:
again, anyway we can just inherit the version? or even inherit the DST file path entirely?